app/models/initiative.rb

```diff
@@ -41,5 +41,8 @@ class Initiative < ApplicationRecord
   has_one :parent, class_name: 'Initiative', foreign_key: 'parent_id'
 
   validates :source, presence: true
-  validates :title, presence: true
+  validates_with InitiativeHasManagerValidator
+  before_create do
+    self.title = self.title || "Anonymous initiative"
+  end
```

Comment:
The titles seem super important for the initiatives so I think they should validate that they are present and have a uniq constraint

Reply:
I think you're right, I'll piggyback it on this PR!